starknet_py/cairo/serialization/data_serializers/felt_serializer_test.py

```diff
@@ -0,0 +1,72 @@
+from typing import cast
+
+import pytest
+
+from starknet_py.cairo.serialization.data_serializers.felt_serializer import (
+    FeltSerializer,
+)
+from starknet_py.cairo.felt import FIELD_PRIME
+from starknet_py.cairo.serialization.errors import (
+    InvalidValueException,
+    InvalidTypeException,
+)
+
+
+@pytest.mark.parametrize(
+    "value",
+    [0, 1, 322132123, FIELD_PRIME - 1],
+)
+def test_valid_felt_values(value):
+    print(value)
+    serialized = FeltSerializer().serialize(value)
+    deserialized = FeltSerializer().deserialize([value])
+
+    assert deserialized == value
+    assert serialized == [value]
+
+
+def test_valid_shortstring():
+    with pytest.warns(DeprecationWarning, match="Serializing shortstrings"):
+        serialized = FeltSerializer().serialize(
+            # We use cast because we don't want to show that strings are accepted in types
+            cast(int, "string shorter than 32 chars")
+        )
+        assert serialized == [
+            0x737472696E672073686F72746572207468616E203332206368617273
+        ]
+
+
+def test_invalid_shortstrings():
+    with pytest.raises(InvalidValueException, match="Expected an ascii string"):
+        FeltSerializer().serialize(cast(int, "õ"))
+
+    with pytest.raises(
+        InvalidValueException, match="cannot be longer than 31 characters"
+    ):
+        FeltSerializer().serialize(cast(int, "12345678901234567890123456789012"))
```

Comment:
String multiplication 💪 
```suggestion
        FeltSerializer().serialize(cast(int, "a" * 32))
```